packages/sensors/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -4,12 +4,19 @@
     <uses-permission android:name="android.permission.INTERNET"/>
 
     <application android:name="io.flutter.app.FlutterApplication" android:label="sensors_example" android:icon="@mipmap/ic_launcher">
-        <activity android:name=".MainActivity"
+        <activity android:name=".EmbeddingV1Activity"
                   android:launchMode="singleTop"
                   android:theme="@android:style/Theme.Black.NoTitleBar"
                   android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|layoutDirection"
                   android:hardwareAccelerated="true"
                   android:windowSoftInputMode="adjustResize">
+        </activity>
+        <activity android:name=".MainActivity"
+            android:launchMode="singleTop"
```

Comment:
FYI @cyanglaz @amirh @mklim @collinjackson @bparrishMines @kroikie I don't think we require the `singleTop` launch mode any more. If you discover the opposite, please let me know, but it is probably best not to include that in the example unless it is actually required. From an add-to-app perspective, single top is a problem.